pandas/_libs/reshape.pyx

```diff
@@ -2,7 +2,9 @@ import cython
 from cython import Py_ssize_t
 
 from numpy cimport (int8_t, int16_t, int32_t, int64_t, uint8_t, uint16_t,
-                    uint32_t, uint64_t, float32_t, float64_t)
+                    uint32_t, uint64_t, float32_t, float64_t, ndarray)
+import numpy
```

Comment:
`import numpy as np`

Reply:
Assuming you use `is_array` as suggested above, you'll need to do `cnp.import_array()` here